### ISSUE TITLE
DEV:LOG:AddToStore will use DoT accepted DateTimeStamp (RTC HIGH PRIORITY)

### ISSUE THREAD

**Issue by TessForGithub2** [hashmapinc/Drillflow#615]
DoT was not complaining about the date/timestamps used by AddToStore, but just sending back the wrong information. The solution is to conform AddToStore date/timestamps to those used by DoT.

This card is connected to #610

**Comment by shehzadsidi** [comment]
Assigning to Such since the changes in GetFromStore

**Comment by shehzadsidi** [comment]
Assigning to Such since the changes in GetFromStore

**Comment by shehzadsidi** [comment]
Assigning to Sukhe since the changes in GetFromStore

**Comment by TessForGithub2** [comment]
UPDATE: After a conversation with Sukhe, I determined that there was 100% fidelity between what AddToStore was providing to DoT and what DoT was returning to GetFromStore. DoT did not throw an error with the date/timestamp as provided by AddToStore. Plus, DoT did not return garbled or "messed up" date/timestamps. Additionally, by examining the schema, I determined that the formats of the date/timestamp provided by the client (in our case, our test client) was valid in format. The GetFromStore must handle the date/timestamp returned correctly. I provided this link to Sukhe to help him with any formatting issues:

https://howtodoinjava.com/java/date-time/xmlgregoriancalendar-date-string-example/

It is my understanding that the data is now being returned correctly.
